cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/creating-a-multicomponent-application-with-odo.adoc

```diff
@@ -13,7 +13,7 @@ This example describes how to deploy a multicomponent application - a shooter ga
 .Prerequisites
 
 * `{odo-title}` is installed.
-* You have a running cluster. Developers can use link:https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/[CodeReady Containers (CRC)] to deploy a local cluster quickly.
+* You have a running cluster. Developers can use link:https://access.redhat.com/documentation/en-us/red_hat_openshift_local/[{openshift-local-productname}] to deploy a local cluster quickly.
```

Comment:
It looks like the four CLI reference pages that are updated in this PR are not live in any of the topic maps. Given this, I can't review a preview of how `{openshift-local-productname}` would render in those cross-references if those pages were made live. But, given the use of the same attribute in `welcome/oke_about.adoc`, I think they look ok.